File: Task1/color.cpp
#include "color.h"

color ColorFromNumber(int& colorNumber) {
    switch (colorNumber) {
        case 1:
            return red;
        case 2:
            return orange;
        case 3:
            return yellow;
        case 4:
            return green;
        case 5:
            return lightblue;
        case 6:
            return darkblue;
        default:
            return purple;
    }

}

File: Task3/shapes/color.py
import enum


class Color(enum.Enum):
    red = 0
    orange = 1
    yellow = 2
    green = 3
    light_blue = 4
    dark_blue = 5
    purple = 6


def get_color_from_int(number: int) -> Color:
    for color in Color:
        if color.value == number:
            return color
    return Color.red
